core/modules/css/KvInterfaceImplMySql.cc

```diff
@@ -568,9 +568,11 @@ std::string KvInterfaceImplMySql::dumpKV() {
         if (row[1].first == nullptr or *row[1].first == '\0') {
             result += "\\N";
         } else {
+/*
             if (strchr(row[1].first, '\n') != nullptr) {
                 throw CssError("KvInterfaceImplMySql::dumpKV - value contains newline");
             }
+*/
```

Comment:
Well, the output is not very human-friendly so it has to be for something else :) I think qserv-admin was supposed to have a RESTORE command which would read DUMP output, but it was never implemented.